XX.md

```diff
@@ -0,0 +1,67 @@
+# NIP XXXX: Read Status Tracking via Salted Hash Keys and Compressed 64-bit Roaring Bitmaps
+
+**Abstract:**  
+Track message read status by publishing replaceable events (kind 30050) containing a compressed 64-bit Roaring Bitmap of read-event keys.
+
+**Specification:**
+
+1. **Secret Salt:**
+  
+  - Generate a random 32-byte salt and encrypt it using NIP‑44.
```

Comment:
How should this random salt be shared across clients?

Reply:
see https://github.com/prolic/nips/blob/roaring/XX.md?plain=1#L